tests/test_app_extended_coverage.py

```
@@ -12,7 +12,21 @@
 from fastapi.testclient import TestClient
 from starlette.types import ASGIApp
 
-from app import app
+import sys
+import os
```

Ruff (0.13.1)

Redefinition of unused `os` from line 7

Remove definition: `os`

(F811)